src/rules/at-import-partial-extension-blacklist/README.md

```diff
@@ -0,0 +1,70 @@
+# at-import-partial-extension-blacklist
+
+Specify blacklist of disallowed file extensions for partial names in `@import` commands.
+
+```scss
+@import "path/to/file.scss"
+/**                  ↑
+ *                   Blacklist of these */
```

Comment:
Minor niggle, but the the convention is keep things as far left as possible.

If we don't expect the dot to be included then I suggest pointing the arrow at the "s" or "scss", and then aligning the bottom of arrow with the last "e" in "these".